src/Microsoft.AspNetCore.Localization/Internal/RequestCultureProviderLoggerExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved. 
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.Localization.Internal
```

Comment:
This is in the wrong namespace. The namespace (in general) needs to be based on the assembly's main namespace. Maybe it should be `Microsoft.AspNetCore.Localization.Internal`?